skyportal/handlers/api/candidate/candidate.py

```diff
@@ -853,22 +861,26 @@ def get(self, obj_id=None):
             ]
 
             if isinstance(group_ids, str):
```

Comment:
since filters are subsets of groups, shouldn't we first check if filter_ids are specified and if so use those, and THEN check if groups are specified and if so get all filters for those?

Reply:
This is just a comment about the logic but let's keep it as is since that's how it's implemented. Maybe worth opening an issue for this one. Anyways let's just not make that change in that PR.